docs/web-apps/automated-testing/cypress/yaml/v1alpha.md

```diff
@@ -898,14 +898,14 @@ mode: 'sauce'
 Provides details related to the Cypress test configuration that are relevant for this test suite.
 
 ```yaml {5}
```

Comment:
Isn't the `{5}` bringing any side-effects ?

Reply:
Huh, the `{5}` is to highlight a line in the snippet. 🤯

Reply:
https://docs.saucelabs.com/web-apps/automated-testing/cypress/yaml/v1alpha/#config